src/System.Drawing.Primitives/src/System/Drawing/Size.cs

```diff
@@ -74,6 +74,62 @@ public Size(int width, int height)
         public static Size operator -(Size sz1, Size sz2) => Subtract(sz1, sz2);
 
         /// <summary>
+        /// Multiplication operator that multiplies <see cref="Size"/> by an <see cref="int"/> producing <see cref="Size"/>.
```

Comment:
Can you remove "<foo> operator that" from all of these XML comments? It is a little redundant. This can just be `Multiplies a <see cref="Size"/> by an <see cref="int"/>, producing <see cref="Size"/>.`